src/hooks/useOperationModal/Modal/WithdrawForm/index.tsx

```diff
@@ -127,17 +127,17 @@ export const WithdrawFormUi: React.FC<WithdrawFormUiProps> = ({
           onClick: handleRightMaxButtonClick,
         }}
         hasError={!!formError && Number(formValues.amountTokens) > 0}
-        description={
-          <Trans
-            i18nKey="operationModal.withdraw.withdrawableAmount"
-            components={{
-              White: <span css={sharedStyles.whiteLabel} />,
-            }}
-            values={{ amount: readableWithdrawableAmountTokens }}
-          />
-        }
       />
 
+      <LabeledInlineContent
+        label={t('operationModal.withdraw.withdrawableAmount')}
+        css={sharedStyles.getRow({ isLast: true })}
```

Comment:
You could avoid the javascript and do this in pure css using last child selector